### PR TITLE
AWS Template improvements (input prompt for configs, default to Redshift)

### PR DESCRIPTION
**Pull request by tsotnet**
AWS Template improvements (input prompt for configs, default to Redshift)

Signed-off-by: Tsotne Tabidze <tsotne@tecton.ai>

**What this PR does / why we need it**: Currently AWS template uses dynamodb online store and file offline store. We're now changing the offline store to Redshift. However, Redshift needs to be configured by the user to make things work (e.g. cluster_id, database name, etc.). We don't have a mechanism for that. This PR adds [python-inquirer](https://github.com/magmax/python-inquirer) library for user prompt during aws template bootstrap, which is used for configuring the feature_store.yaml. Here's the new flow in action:

image

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Allow configuring online & offline stores during AWS template, default to Redshift offline store
```

### CODE REVIEW

File: sdk/python/feast/templates/aws/bootstrap.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import inquirer
```
Comment (woop) [inline sdk/python/feast/templates/aws/bootstrap.py:1]: Out of curiousity, is there anything `inquirer` gives us that `Click` doesn't?

**Review by woop — APPROVED** [review]

File: sdk/python/setup.py [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "tenacity>=7.*",
     "toml==0.10.*",
     "tqdm==4.*",
+    "inquirer==2.7.0",
```
Comment (achals) [inline sdk/python/setup.py:60]: Any reason we're pinning to this version and not a range?

File: sdk/python/feast/templates/aws/bootstrap.py (hunk 1 above)
Comment (achals) [reply sdk/python/feast/templates/aws/bootstrap.py:1]: +1 to fewer deps if possible.

**Review by achals — APPROVED** [review]
/lgtm

**Pull request merged**